### PR TITLE
Feat/txn state proof pk

### PR DESCRIPTION
**Pull request by vuvoth**
## Proposed Changes

- Add txn StateProofPK
- Uint tests. 
- Update changelog

### CODE REVIEW

File: packages/runtime/src/lib/txn.ts [hunk 1]
```diff
@@ -185,6 +185,13 @@ export function txnSpecByField(
 			break;
 		}
 
+		case "StateProofPK": {
+			// When I run teal debug, "StateProofPK" always return 64 zero bytes.
+			// so I set up 64 zero bytes is default value of StateProofPK
```
Comment (ratik21) [inline packages/runtime/src/lib/txn.ts:190]:
```suggestion
			// While running teal debugger, "StateProofPK" always return 64 zero bytes.
			// so we set up 64 zero bytes as default value of StateProofPK
```

Comment (ratik21) [reply packages/runtime/src/lib/txn.ts:190]: Don't talk in first person, in code comments :sweat_smile:

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,9 +19,10 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 - Add Runtime.getAppByName(appName): get app by name declared in appDefinition.
 - For Algob.balanceOf(deployer, accountAddr, assetID) if assetID is undefined then function will return ALGO account balance.
 - Add new example [Trampoline](https://github.com/algorand-devrel/demo-avm1.1/tree/master/demos/trampoline)
-- Runtime: add `Txn LastLog` opcode.
+- Runtime: add `Txn LastLog` opcode(tealv6).
 - Runtime: better warning/error when deploying ASA. Throws an error when ASA definition is wrong or when ASA is not found in asa.yaml, eg when Runtime needs to query ASA.
 - Add `Runtime.getAppByName(appName)`. We can get application in Runtime now.
+- Runtime: add `Txn StateProofPK` opcode(tealv6).
```
Comment (ratik21) [inline CHANGELOG.md:25]: Probably better to have a separate header for `@algo-builer/runtime`

**Review by ratik21 — APPROVED** [review]

File: packages/runtime/src/lib/txn.ts (hunk 1 above)
Comment (vuvoth) [reply packages/runtime/src/lib/txn.ts:190]: Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (vuvoth) [reply CHANGELOG.md:25]: Good suggestion!

Comment (vuvoth) [reply CHANGELOG.md]: Done!

**Pull request merged**